src/hotspot/cpu/x86/stubGenerator_x86_64.cpp

```diff
@@ -61,4889 +48,4921 @@
 #if INCLUDE_JFR
 #include "jfr/support/jfrIntrinsics.hpp"
 #endif
+#include "stubGenerator_x86_64.hpp"
```

Comment:
This should be in alphabetical order prior to the conditional includes.

Reply:
Fixed.